src/internal/connector/sharepoint/collection_test.go

```diff
@@ -106,9 +106,32 @@ func (suite *SharePointCollectionSuite) TestRestoreListCollection() {
 	account, err := a.M365Config()
 	require.NoError(t, err)
 
-	service, err := createTestService(account)
+	col, err := collectPages(
+		ctx,
+		account,
+		nil,
+		account.AzureTenantID,
+		siteID,
+		nil,
+		&MockGraphService{},
+		control.Defaults(),
+	)
+	assert.NoError(t, err)
+	assert.NotEmpty(t, col)
+}
+
+// TestRestoreListCollection verifies Graph Restore API for the List Collection
+func (suite *SharePointCollectionSuite) TestRestoreListCollection() {
+	ctx, flush := tester.NewContext()
+	defer flush()
+
+	t := suite.T()
+	siteID := tester.M365SiteID(t)
+	a := tester.NewM365Account(t)
+	account, err := a.M365Config()
```

Comment:
These values are set by multiple tests in the suite.  Could be moved to the suite setup for cleanliness.

Reply:
Resolved within [7846c99](https://github.com/alcionai/corso/pull/2178/commits/7846c99c9a50df03dea6810449487d62982b3db6)